stablehlo/reference/Ops.cpp

```diff
@@ -156,6 +156,27 @@ Tensor makeSplat(ShapedType type, const Element &initValue) {
   return result;
 }
 
+SmallVector<Tensor> split(const Tensor &input, int64_t groupSize,
```

Comment:
`numResults` to match how this function is defined in the spec?

Reply:
Should I rename the remaining params to match the spec?
`split(x: Value, num_results: Value, axis: Value) -> Value`

Reply:
Sounds good